src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1454,8 +1464,7 @@ const mesh::Mesh &SolverInterfaceImpl::mesh(const std::string &meshName) const
       return *context->mesh;
     }
   }
-  PRECICE_ERROR("Participant \"" << _accessorName
-                                 << "\" does not use mesh \"" << meshName << "\"!");
+  PRECICE_ASSERT(false, "Participant \"" << _accessorName << "\" does not use mesh \"" << meshName << "\".");
```

Comment:
Why `PRECICE_ASSERT(false,...)` and not just `PRECICE_ERROR(...)`?

Reply:
This is only used in integration tests and means that the setup is incorrect. A user will never run into this code path.
Thus I would say that this should be an assertion.